app/assets/javascripts/admin/addon/templates/user-index.hbs

```diff
@@ -641,6 +641,17 @@
     <div class="field">{{i18n "user.invited.days_visited"}}</div>
     <div class="value">{{html-safe model.days_visited}}</div>
   </div>
+  <div class="display-row post-edits-count">
+    <div class="field">{{i18n "user.invited.post_edits_count" }}</div>
```

Comment:
The `user.invited.post_edits_count` key feels a little off to me because we're putting it under `invited` when there is no relations to invites.

Reply:
Will move it to admin.user